### PR TITLE
[RPA-1890] Change api/processes to scheduler

### PR DESCRIPTION
**Pull request by Aeri4a**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-desktop/src/action/general/general.action-handler.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export default class GeneralActionHandler extends StatelessActionHandler {
     constructor(
         @Inject(forwardRef(() => RuntimeService))
         private runtimeService: RuntimeService,
+        private readonly storageService: StorageService,
```
Comment (marcin-depcik) [inline runbotics/runbotics-desktop/src/action/general/general.action-handler.ts:24]: I think you need to import ConfigModule inside CoreModule

File: runbotics/runbotics-desktop/src/action/general/general.action-handler.ts [hunk 2]
```diff
@@ -44,8 +45,9 @@ export default class GeneralActionHandler extends StatelessActionHandler {
     async startProcess(
         request: DesktopRunRequest<GeneralAction.START_PROCESS, StartProcessActionInput>
     ): Promise<StartProcessActionOutput> {
-        const response = await orchestratorAxios.get<IProcess>(
-            `/api/processes/${request.input.processId}`,
+        const tenantId = this.storageService.getValue('tenantId');
```
Comment (marcin-depcik) [inline runbotics/runbotics-desktop/src/action/general/general.action-handler.ts:48]: I know that, we have such implementation, but I don't think this is a good approach of keeping such information like `tenantId`. Maybe we should think about changing it in the future

**Review by marcin-depcik — APPROVED** [review]

**Review by marcin-depcik — COMMENTED** [review]
bump version

File: runbotics/runbotics-desktop/src/action/general/general.action-handler.ts (hunk 2 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-desktop/src/action/general/general.action-handler.ts:48]: Good point, if it's stored in local storage there's a risk someone can exploit that to access resources of another tenant

File: runbotics/runbotics-desktop/src/core/websocket/listeners/process.listener.ts [hunk 3]
```diff
@@ -89,8 +90,11 @@ export class ProcessListener {
             }
 
             const { processId, input, ...rest } = data;
-            const process = await orchestratorAxios
-                .get<IProcess>(`/api/processes/${processId}`, { maxRedirects: 0 })
+            const tenantId = this.storageService.getValue('tenantId');
```
Comment (star-in-the-jar) [inline runbotics/runbotics-desktop/src/core/websocket/listeners/process.listener.ts:93]: If there's a risk of overwriting this value by a user, we need a bugfix task in #tobediscussed

**Review by star-in-the-jar — CHANGES_REQUESTED** [review]
+version

File: runbotics/runbotics-desktop/src/core/websocket/listeners/process.listener.ts (hunk 3 above)
Comment (marcin-depcik) [reply runbotics/runbotics-desktop/src/core/websocket/listeners/process.listener.ts:93]: I created task for this

**Pull request merged**